serverlessworkflow/modules/ROOT/pages/getting-started/java-embedded-workflows.adoc

```diff
@@ -0,0 +1,104 @@
+= Workflow embedded execution in Java 
+
+This guide show cases how to execute a link:{spec_doc_url}[CNCF Serverless Workflow] definition using a standard java virtual machine and a small set of Maven dependencies. Therefore, it is assumed you are fluent both in Java and Maven. 
+The workflow definition to be executed can be read from a .json or .yaml file or programatically defined using the {product_name} fluent API. 
+
+[[embedded-file-quick-start]]
+== Hello world (using existing definition file)
+
+First step is to setup an empty Maven project with the following dependency. 
+
+[source,xml]
+----
+<dependency>
```

Comment:
@ricardozanini Yes, the renaming from kogito to sonataflow is far from completed

Reply:
The rename will be done with https://github.com/apache/incubator-kie-kogito-runtimes/issues/3447